docker-compose.yml

```diff
@@ -29,5 +29,13 @@ services:
       - POSTGRES_USER=${DATABASE_USER}
       - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
 
+  # Email Server and Web Interface for viewing and testing emails during development
+  maildev:
+    image: maildev/maildev
+    container_name: maildev
+    ports:
+      - "1025:1025" # SMTP protocol port
```

Comment:
A good practice to move sensitive or environment-specific configuration data like ports to a .env.
Like this for example:
```
      - "${MAILDEV_SMTP_PORT}:1025" # SMTP protocol port
      - "${MAILDEV_UI_PORT}:1080"   # UI port
```

Reply:
Usually, the .env file is used to store environment variables, not to define the configuration of services.

Reply:
Done!